contents/blog/aarrr-how-to-build-pirate-funnel-posthog-with-posthog.md

```diff
@@ -0,0 +1,157 @@
+---
+date: 2021-09-23
```

Comment:
@joethreepwood - what date do you want on this blog post?

Reply:
I'll correct the date when we publish it. At the moment, I'd like to confirm with the author if we're OK to publish this still and how we should credit it, so it's on hold until then.

Reply:
> I'd like to confirm with the author if we're OK to publish this still and how we should credit it, so it's on hold until then

💯  👏

Reply:
Just spoke with Hanna. She’s happy to be credited as a contributor and the door is potentially open for future contributions too.